src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -74,6 +90,13 @@
  *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_IF, LITERAL_ELSE&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * if (true) {      // OK
+ *     return true;
+ * } else           // violation, single-line statements not allowed without braces
+ *     return false;
+ * </pre>
```

Comment:
I really mean to have the same code block in all examples.
This will make it clear to user that config like this  do:
```
 * while (counter &lt; 10)          // OK, single-line statements not allowed without braces
 *     ++count;
```

and it will become very visual for user.

please update java code in default to have a bit more cases like

```
if (obj.isValid()) return true;      // OK
```

```
switch (num) {
  case 1: counter++; break;      // OK
``` 

```
while (value.incrementValue() < 5);             // OK
for(int i = 0; i < 10; value.incrementValue()); // OK
```

ONE MORE TIME: 
same java code for "all", only config is changing, and in java code only trailing comment are changing.

-----
ATTENTION: skip usage of the same for `To configure the check to lambdas: ` it might be too messy.

Reply:
I added most of the default config to to other cases as well. I didn't add the example of `if-else` to checks where tokens used were not related and the scope of check was different.

Reply:
please use the same java code block for:
`To configure the check for if and else blocks: `
`To configure the check to allow case, default single-line statements without braces: `
`To configure the check to allow loops (while, for) with empty bodies: `

only `To configure the check to lambdas:` should stay as is.

Reply:
not done, please use the same java code for all except for `To configure the check to lambdas`